### ISSUE TITLE
Plot log-odds-ratio

### ISSUE THREAD

**Issue by dariober** [mskcc/facets#65]
Hello again - I have a question about the log-odds-ratio reported by `emcncf` (specifically in `emcncf(xx)$cncf$mafR`) vs the one plotted by `plotSample` (middle plot). 

I see from the code of plotSample that mafR is transformed as `sqrt(mafR)` thus creating an inconsistency between plot and output data. 

Is there any reason for doing this? How should the sqrt(mafR) be interpreted?

Thank you!
Dario

**Comment by veseshan** [comment]
Because we don't know if the reference allele is in the maternal or paternal chromosome the log-odds-ratio for any het SNP can be positive or negative. But the square of it has a constant mean in a segment. What we call mafR in the output is an estimate of squared log-odds-ratio. Hence square root in plotSample. I will add this to the documentation.

Venkat

**Comment by dariober** [comment]
Hi- thanks for the quick reply! 
If I understand correctly, the quantity of interest is actually sqrt(mafR) and indeed this is what plotSample shows. So, instead of editing the documentation, wouldn't it be better to change the output from the current one, mafR, to directly report `sqrt(abs(mafR))`? (Of course, this would be a major change from one version of facets to another)

**Comment by veseshan** [comment]
Makes sense to return sqrt(abs(mafR)) since that is the underlying allelic log-odds ratio. I need to add prominent messaging so that users don't miss it. Thanks.

**Comment by ZooJooFoo** [comment]
Hi-I have a question about “plot the logOR data and mafR”  with  command： 

“jseg$valor[is.finite(jseg$cnlr)”

It seems use logR（cnlr）to judge logOR（valor）？
Should it be  “jseg$valor[is.finite(jseg$valor)” ？

Looking forward to your explanation ！

Thanks！
Yang

**Comment by ZooJooFoo** [comment]
Thanks for your reply！
I didn't think it through before，and i got it now！

Thanks again！
Yang
